src/main/java/com/philvarner/clamavj/ScanResult.java

```diff
@@ -12,12 +12,13 @@
     public static final String STREAM_PATH = "stream";
     public static final String OK_SUFFIX = "OK";
     public static final String FOUND_SUFFIX = "FOUND";
+    public static final String ERROR_SUFFIX = "ERROR";
 
     public static final String RESPONSE_SIZE_EXCEEDED = "INSTREAM size limit exceeded. ERROR";
```

Comment:
maybe remove these two unused constants?

Reply:
I removed `RESPONSE_ERROR_WRITING_FILE` since it is no longer used.  `RESPONSE_SIZE_EXCEEDED` is used in a unit test, and is in the original repo, so I'm going to retain it.